static/_redirects

```diff
@@ -502,3 +502,5 @@
 /docs/setup/cluster-large/     /docs/setup/best-practices/cluster-large/   301
 /docs/setup/node-conformance/     /docs/setup/best-practices/node-conformance/   301
 /docs/setup/certificates/     /docs/setup/best-practices/certificates/   301
+
+/docs/tasks/tools/install-minikube/     https://minikube.sigs.k8s.io/docs/start/     307
```

Comment:
@sftim is this correct? The deploy preview 404s, but I'm not sure if that's an artifact of it being the deploy preview, or if this is not the correct way to do a 307 redirect with this site.

Deploy preview: https://deploy-preview-24416--kubernetes-io-master-staging.netlify.app/docs/tasks/tools/install-minikube/

Reply:
@nate-double-u,
Not sure what is happening with the page redirect. Have you tried to use 301 instead of 307?

Reply:
301 would very likely work -- but I won't know till the next time I push and let the deploy-preview build (the redirects don't seem to work locally for me, but do on the preview). When I update for the other feedback provided I'll try it. 
Even if it's not the 307 that @sftim asked for, I think some redirect is better than none.

Reply:
See test PR #24458

Reply:
Looks like 302 works in that PR (https://deploy-preview-24458--kubernetes-io-master-staging.netlify.app/docs/tasks/tools/install-minikube/), so the issue maybe with the setup for the 307 redirect itself.

Reply:
Digging a bit deeper, looks like Netlify doesn't support 307 redirects:
- https://github.com/netlify/netlify-playground/issues/35
- https://community.netlify.com/t/307-temporary-redirect-does-not-work-anymore/15516

Suppose I'll change it to 302 then.

Reply:
The preview just updated with my last push, the 302 redirect works now for https://deploy-preview-24416--kubernetes-io-master-staging.netlify.app/docs/tasks/tools/install-minikube/